acp/source_hub_client.go

```diff
@@ -94,6 +90,7 @@ type sourceHubBridge struct {
 	client sourceHubClient
 }
 
+var _ ACP = (*sourceHubBridge)(nil)
```

Comment:
todo: Guessing you accidently duplicated this, please remove.

Reply:
This was marked resolved but is still not done, so unresolving.

Reply:
i believe it should be done, unless i'm missing something